lib/scheduler/variable_looping_vus.go

```diff
@@ -0,0 +1,59 @@
+package scheduler
+
+import (
+	"fmt"
+
+	"github.com/loadimpact/k6/lib/types"
+	null "gopkg.in/guregu/null.v3"
+)
+
+const variableLoopingVUsType = "variable-looping-vus"
+
+// Stage contains
+type Stage struct {
+	Duration types.NullDuration `json:"duration"`
+	Target   null.Int           `json:"target"` // TODO: maybe rename this to endVUs?
```

Comment:
targetVU, `end` seem strange in this case although given we have `startVU` maybe it is okay

Reply:
Yeah, same issue as the `stages` in the `VariableArrivalRateConfig` above... `endVUs` seems clearer than `target` to me, but not by much

Reply:
Leaving it `target` for now